store/sdk/src/main/java/org/apache/carbondata/sdk/file/AvroCarbonWriter.java

```diff
@@ -445,17 +445,13 @@ private static Field prepareFields(Schema.Field avroField) {
         if (logicalType instanceof LogicalTypes.Date) {
```

Comment:
Int can have "Time (millisecond precision)" logical type annotation. So, please leave a comment here about logical type  handling in carbon. I think we just leave int as it is.

Same way, add remarks for time-micros also.  
It would be good if we can summarize the logical types vs carbon data types mapping in the converter class it will beusefull for future reference and reviews.

Reply:
Added a comment